pos_save_client_error_fix/static/src/js/pos_save_client_error_fix.js

```diff
@@ -0,0 +1,46 @@
+/* © 2016 Diagram Software S.L.
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+(function ($) {
+    "use strict";
+    openerp.pos_save_client_error_fix = function (instance, module) {
```

Comment:
I think that the header is unnecessary. (function($)) and jQuery at the end.

See more lighter header here : 
https://github.com/OCA/pos/blob/8.0/pos_customer_required/static/src/js/pos_customer_required.js#L1